### ISSUE TITLE
I have found some ios 8 specific apis after dragging source code into my project

### ISSUE THREAD

**Issue by superZhen** [SnapKit/SnapKit#141]
Thanks for this great kit.

For example,  here is compile error in my xcode 7 beta6:
'activateConstraints' is only available on iOS application extension 8.0 or newer
'FirstBaseline' is only available on iOS application extension 8.0 or newer
'CenterYWithinMargins' is only available on iOS application extension 8.0 or 
'CenterXWithinMargins' is only available on iOS application extension 8.0 or newer
'TrailingMargin' is only available on iOS application extension 8.0 or newer

do you have any plan to fix those errors?

**Comment by robertjpayne** [comment]
@superZhen hm, I'll check it out for errors. I do have it setup to try and gracefully degrade:

```
    internal override func activate() -> Void {
        if NSLayoutConstraint.respondsToSelector("activateConstraints:") && self.installInfo != nil {
            let layoutConstraints = self.installInfo!.layoutConstraints.allObjects as! [LayoutConstraint]
            if layoutConstraints.count > 0 {
                NSLayoutConstraint.activateConstraints(layoutConstraints)
            }
        } else {
            self.install()
        }
    }
```

**Comment by superZhen** [comment]
so quick reply!  i will wait for the new upgrade. Have a nice day~

**Comment by robertjpayne** [comment]
@superZhen yea I need to fix this for iOS 7.0 and OS X 10.9 support. Sorry it will take a bit because I'm away for this next week.

**Comment by superZhen** [comment]
@robertjpayne, take your time.

**Comment by espiegel** [comment]
@superZhen @robertjpayne Is there any quickfix I can use meanwhile to support ios 7 ?

**Comment by robertjpayne** [comment]
There is no quick fix right now for Swift 2.0 and iOS 7.0. I was away for a week and thus not had time to fix and push.

The only quick fix is to fix the source code yourself when integrating into an iOS 7 target.

**Comment by devxoul** [comment]
@espiegel, you can use `develop` branch that contains https://github.com/SnapKit/SnapKit/pull/143

**Comment by robertjpayne** [comment]
Released with 0.15.0
